src/style.css

```
@@ -1,3 +1,83 @@
-body {
+html {
+color: #000;
```

Use 2 spaces for indentation in your file
Fix it in all cases
```suggestion
  color: #000;
```